capirca/aclgen.py

```diff
@@ -55,53 +55,56 @@
 
 
 FLAGS = flags.FLAGS
-flags.DEFINE_string(
+
+
+def SetupFlags():
```

Comment:
External applications that also use `absl` can hit name conflicts when importing this file. Containing the setup in it's own function allows us to do the setup only when necessary, instead of at import time